### ISSUE TITLE
Change Splitscreen Orientation for Ultrawide? 

### ISSUE THREAD

**Issue by conorshone** [WinterSquire/AlphaRing#54]
Probably a bit niche because of my monitor as I'm using an ultrawide (5120x1440), but wanted to know if it's possible to change the orientation of the splitscreen, so side to side rather than above and below? As at the moment I have 2 very wide but narrow views, where as if the orientation was changed to side to side we'd have a 16:9 screen each. 

Cheers

**Comment by Jipi117** [comment]
Hey, I don't think its niche, that would be a great feature. any person with a dual screen setup can turn 2 screens into an ultra wide screen using a software like nvidia surrond span. That would allow you do play coop 2 player coop on 2 different screen. I am all for it!

**Comment by alwayserik** [comment]
I second this! Please add support for this ASAP!

**Comment by Terminative** [comment]
I would love to have this feature as well!

**Comment by Macrisss** [comment]
Yes, would really love a vertical split option for two player co-op!

**Comment by ginger-mochi** [comment]
Would really like to have this, even for non-ultrawide.
